packages/aws-cdk-lib/aws-iam/README.md

```diff
@@ -369,6 +369,39 @@ new iam.Role(this, 'Role', {
 });
 ```
 
+### Granting an identity permission to assume a role
```

Comment:
Can you change this to "Granting a principal...". That's the more common verbiage.

Reply:
Ah, I forgot I'd modified this.  I changed this to identity based on [this section](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-iam/README.md#principals-vs-identities) of the readme but I think I'm just misreading it.  Will change